src/main/java/com/telefonica/iot/cygnus/backends/ckan/CKANCache.java

```diff
@@ -355,8 +355,10 @@ private void populatePackagesMap(JSONArray packages, String orgName) throws Exce
             LOGGER.debug("Package found in CKAN, now cached (orgName=" + orgName + " -> pkgName/pkgId=" + pkgName
                     + "/" + pkgId + ")");
             
-            // get the resources
-            JSONArray resources = (JSONArray) pkg.get("resources");
+//            // get the resources
+//            JSONArray resources = (JSONArray) pkg.get("resources");
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/e6a6b43f5d2506c4b49669e7f108c6a9e2ae66cc